graphql/schema/types/version.graphql

```diff
@@ -39,19 +39,33 @@ type Version {
   revision: String!
   startTime: Time
   status: String!
+  versionTasks(
```

Comment:
`tasks()`? 
So it's `version.tasks` instead of saying the word version twice?
 what do you think?

Reply:
I initially did this but because `versionTasks` contains `tasks` and `count`, it becomes `version.tasks.tasks` when you want to get the actual tasks 😭... interested to hear what you think about that